### ISSUE TITLE
Fix error in OPA EKS 1.18

### ISSUE THREAD

**Issue by jasonBirchall** [ministryofjustice/cloud-platform#3093]
## Background

An engineer experienced an error when using the OPA fix outlined [here](https://github.com/ministryofjustice/cloud-platform-terraform-opa/releases/tag/0.0.12). 

The error appears when making a change to an ingress resource:

```
Error: cannot patch "prometheus-operator-grafana" with kind Ingress: Internal error occurred: failed calling webhook "webhook.openpolicyagent.org": Post https://opa.opa.svc:443/?timeout=30s: x509: certificate signed by unknown authority
 on .terraform/modules/monitoring/prometheus.tf line 85, in resource "helm_release" "prometheus_operator":
 85: resource "helm_release" "prometheus_operator" {
```

## Proposed user journey

Create a 1.18 cluster - use the latest tag for OPA - observe the error and attempt to resolve by upgrading to EKS 1.19

## Reference

[How to write good user stories](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)

**Comment by vijay-veeranki** [comment]
![Screenshot 2021-07-27 at 12.38.52.png](https://images.zenhubusercontent.com/5cd14e5c7c13c23a96974cd6/d1307a02-6071-4836-8dff-78e24b56dfd5)

**Comment by vijay-veeranki** [comment]
Me and Paul worked on it and we reproduced the issue.

1) We created new EKS cluster (v1.18)
2) Delete opa webhook
"kubectl delete validatingwebhookconfigurations.admissionregistration.k8s.io opa"
3) use the latest OPA version "github.com/ministryofjustice/cloud-platform-terraform-opa?ref=0.0.12"
and apply it.
  OPA validation webhook got re-created and pods got re-created.
4) Run all smoke tests, Ingress test failed.
5) Run just ingress test, we seen the error

`       +Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "webhook.openpolicyagent.org": Post https://opa.opa.svc:443/?timeout=30s: x509: certificate signed by unknown authority`
